benchmarks/MultipleEventMultipleLocalAndSharedListenerBench.php

```diff
@@ -1,12 +1,11 @@
 <?php
 
+declare(strict_types=1);
+
 namespace LaminasBench\EventManager;
 
 use Laminas\EventManager\EventManager;
 use Laminas\EventManager\SharedEventManager;
-use PhpBench\Benchmark\Metadata\Annotations\Iterations;
```

Comment:
These imports keep being removed 🤔

Reply:
Sorry… I'll fix that now. It's probably `optimize imports` in PHPStorm - it frequently messes up imports for annotations